mitodl/ccxcon
# Added doc about how to configure ccxcon with edx

#### What are the relevant tickets?

fixes #142 
#### What's this PR do?

Adds documentation about how to configure CCXCON to work with an EDX instance
#### Where should the reviewer start?

Look at the doc and check that it renders properly
#### How should this be manually tested?

In theory you should make a setup from scratch end verify that these steps are the only required 
#### Any background context you want to provide?

You need a working EDX and a working CCXCON
#### What gif best describes this PR or how it makes you feel?

![](http://i.giphy.com/l41lLpzDnSgGOwCY0.gif)
